packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts

```diff
@@ -361,8 +361,9 @@ describe('plugin-metrics', () => {
         });
       });
 
+      //TODO: The following two skipped tests needs investigation: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-485382
```

Comment:
shouldn't SPARK-485382 be under SPARK-513448? is the plan to fix it before we move webapp to "next"?

Reply:
Since we have changed metrics to use mocha these test are passing. When we move metrics to use jest these tests needs to be looked into.